mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS fs net/minecraft/block/dispenser/DispenserSoundBehaviour
+	FIELD b eventType Z
```

Comment:
Boolean? Type?

Reply:
This is used in a ternary operator between event types 1000 and 1001.

Reply:
Renamed as eventTypeSelector

Reply:
What does each event type represent?

Reply:
I think this name should reflect what the event types are